lib/state/appActions.js

```diff
@@ -62,6 +62,7 @@ appActions.errorStages = {
   STAGE_PICK_FILE : { code: 'E_SELECTING_FILE' , friendlyMessage: 'Error during file selection' },
   STAGE_READ_FILE : { code: 'E_READING_FILE' , friendlyMessage: 'Error trying to read file' },
   STAGE_UPLOAD : { code: 'E_UPLOADING' , friendlyMessage: 'Error uploading data' },
+  STAGE_METADATA_UPLOAD : { code: 'E_METADATA_UPLOAD' , friendlyMessage: 'Error uploading upload metadata'},
```

Comment:
This never gets exposed to the user because of the immediate redirect to the "Your uploader is out-of-date" page. It does look like this (if you put in a 10-second timeout on the redirect at least...):
![screenshot 2015-06-08 16 10 14](https://cloud.githubusercontent.com/assets/1588547/8049477/9cf9d206-0e14-11e5-826a-b075c8777b8f.png)

I was thinking we might want to generalize the same error to other upload metadata validation problems.